pkg/common/security/usergroup_test_resolver.go

```diff
@@ -41,15 +41,19 @@ func GetUserGroupCacheTest() *UserGroupCache {
 // test function only
 func lookup(userName string) (*user.User, error) {
```

Comment:
This test offers the testing-only resolver to other test cases. Hence, could you please move those constants to the beginning of file, and other tests should share it instead of creating a bunch of "const"

Reply:
Thanks for your feedback! I'll move those constants to the beginning of the file.